code/modules/cargo/packs/service.dm

```diff
@@ -159,16 +159,6 @@
 					/obj/item/reagent_containers/food/drinks/drinkingglass/shotglass)
 	crate_name = "kitchen cutlery deluxe set"
 
-/datum/supply_pack/service/replacementdb
```

Comment:
Its nice to be able to get some guns from cargo you know...

Reply:
Always thought the price should have been 2800~ credits by the by,  2200 is just to low.  But again rarely people buy these.

Reply:
People will catch on. I want to nip the overabundance of ballistic firearms in the bud.

Reply:
Im more or fine with this crate getting a heavy nerf rather then flat out removal

Reply:
You can't really nerf infinite firearms. I mean you could nerf the infinite money generatoes in cargo but I don't care about that right now.

Reply:
making it cost more like 3000 is a nerf/balance thing, if you feel its to cheap.